packages/dds/sequence/src/test/overlappingSequenceIntervalsIndex.spec.ts

```diff
@@ -19,6 +19,8 @@ import {
 	createOverlappingSequenceIntervalsIndex,
 } from "../intervalIndex/index.js";
 import { SequenceInterval } from "../intervals/index.js";
+// eslint-disable-next-line import/no-internal-modules
+import { SequenceIntervalClass } from "../intervals/sequenceInterval.js";
```

Comment:
nit: can move this import to the index line above, then no need to disable lint rules.